sdk/highlight-next/bin/clean-dist.sh

```diff
@@ -1,2 +1,5 @@
-printf '%s%s' '"use client";' "$(cat dist/next-client.js)" > dist/next-client.js
-printf '%s%s' '"use client";' "$(cat dist/next-client.cjs)" > dist/next-client.cjs
+sed -i '/"use client";/d' dist/next-client.js
```

Comment:
yeah, this doesn't work
```
 ✘  ~/w/h/s/highlight-next   vadim/filters   sed -i '/"use client";/d' dist/next-client.js
sed: 1: "dist/next-client.js": extra characters at the end of d command
```

any other alternatives to sed? otherwise you can make the command work on both mac and non-mac if we get the sed syntax right. now sure how tho

Reply:
I'm going to try this fix:
https://stackoverflow.com/questions/26081375/bsd-sed-extra-characters-at-the-end-of-d-command